app/views/candidate_mailer/nudge_unsubmitted.text.erb

```diff
@@ -6,7 +6,7 @@ If you’re ready, [sign in to your account to submit your application now](<%=
 
 A teacher training adviser could help with your application, if something is holding you back from submitting it.
 
-Our advisers used to be teachers, so they can talk to you about teacher training and teaching as a career.
+Our advisers have years of teaching experience, so they can talk to you about teacher training and teaching as a career.
```

Comment:
I've changed my mind. I think with the line above, it would sound a bit odd to say:

__A teacher training adviser could help with your application, if something is holding you back from submitting it._

_Our advisers can talk to you about teacher training and being a teacher._

So I'd combine everything and say:
A teacher training adviser could help with your application, if something is holding you back from submitting it. They can talk to you about teacher training and being a teacher.